src/time_stream/enums.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class DuplicateOption(Enum):
```

Comment:
shall we add in the other Enums aswell (some in `aggregation` and `relationships` modules)